Frontend/ui-customer/src/MerchantConfigs/Types.purs

```diff
@@ -65,6 +65,7 @@ type AppConfigCustomer a =
   , shareAppConfig :: ShareAppConfig
   , homeScreen :: HomeScreen
   , locationTagBar :: LocationTagBarConfig
+  , enableReAllocation :: Boolean
```

Comment:
Add this in feature block

Reply:
what is feature block ?

Reply:
<img width="502" alt="Screenshot 2023-12-28 at 10 05 23 AM" src="https://github.com/nammayatri/nammayatri/assets/113873457/c94ac1c8-111b-4246-b78f-5ba4ff332256">
Check this part.